src/solvers/LevenbergMarquardt.jl

```diff
@@ -35,6 +35,10 @@ then the keyword `jacobian_tangent_basis` below is ignored
   a functor inheriting from [`StoppingCriterion`](@ref) indicating when to stop.
 * `expect_zero_residual` – (`false`) whether or not the algorithm might expect that the value of
   residual (objective) at mimimum is equal to 0.
+* `η` – parameter of the algorithm, the higher it is the more likely the algorithm will be
+  to reject new proposal points
```

Comment:
This is a bit vague, can we make this more precise?

Reply:
This is the same docs we have for `LevenbergMarquardtState` actually. I think it would be enough to add a range of allowed values? For example `0 < η < 1`.

Reply:
Then we should improve that one as well, this here is really vague and at least for me it does not help what the parameter does. But sure a range is also great.

Reply:
If we add the bound be rephrase it to the rate at which it is rejected or so? Is it deterministic? Then we should write that it is the part of acceptance of a certain value we check?